languages/elixir/exercises/concept/pattern-matching/.docs/instructions.md

```diff
@@ -0,0 +1,44 @@
+In this exercise you will convert the output of two different date functions into strings.
+
+### 1. Define the `DateString.from_tuple/1` function
+
+The Erlang [:calendar.universal_time/0](http://erlang.org/doc/man/calendar.html#universal_time-0) function can be used by Elixir to return the current UTC datetime as a tuple.
```

Comment:
I was slightly confused by the use of dates/time here, as that is not a dependency of this exercise according to the design. Either it should be introduced in this exercise (which I think would not be very sensible) or it should be added as a dependency. That said, the previous comment mentioned coming up with a theme, so it might be best to focus on that first.

Reply:
Dates here are beside the point here: they are just a convenient way to come up with shapes of data (here, nested tuples or maps).  We could just as easily make up random shapes here to practice pattern matching on, but I think using the output of real date functions feels more concrete.

Reply:
> Dates here are beside the point here: they are just a convenient way to come up with shapes of data

Well, to the student they won't be "beside the point," as the student will have to work with them. 

> I think using the output of real date functions feels more concrete.

I understand that, but either a prerequisite should be added for working with dates, or else a different type should be used.